### PR TITLE
feat(networks): describe which ports are opened on our front-servers

### LINKED ISSUE #2080: [Network] Explicit which ports are open on our fronts servers
On this page: https://doc.scalingo.com/platform/internals/network

Scalingo only have these ports opened: 80 and 443

Also add a warning (maybe in another page) that if the customer use Cloudflare Proxy these ports will be open: `80`, `443`, `8080`, `8443` from the perspective of the customer website

https://developers.cloudflare.com/fundamentals/get-started/reference/network-ports/

### PR DESCRIPTION
**Pull request by ipfaze**
fix #2080

### CODE REVIEW

**Comment by ipfaze** [comment]
I added the warning on the same page but I was thinking maybe added it here:
https://doc.scalingo.com/platform/app/cloudflare-scalingo-app#setup-of-your-cloudflare-account

WDYT?

**Comment by brandon-welsch** [comment]
> I added the warning on the same page but I was thinking maybe added it here: https://doc.scalingo.com/platform/app/cloudflare-scalingo-app#setup-of-your-cloudflare-account
> 
> WDYT?

Yes you can use partial content to have only one content (to avoid to duplicate) and use the partial in the 2 pages.

For example: https://github.com/search?q=repo%3AScalingo%2Fdocumentation%20dbclient_fetcher.md&type=code

File: src/_posts/platform/internals/2000-01-01-network.md [hunk 1]
```diff
@@ -56,6 +56,11 @@ output IPs, please refer to our provider documentation
 [here](https://docs.outscale.com/en/userguide/OUTSCALE-Public-IP-Addresses.html).
 Our `osc-secnum-fr1` region is hosted on Outscale `cloudgouv-eu-west-1` region.
 
+## Open Ports
+Scalingo front servers only have ports `80` and `443` opened.
```
Comment (brandon-welsch) [inline src/_posts/platform/internals/2000-01-01-network.md:60]:
```suggestion
Scalingo front-end servers have the following ports open: `80` and `443`.
```

nitpick: I prefer this one WDYT?

**Review by brandon-welsch — APPROVED** [review]
LGTM Thanks ❤️

**Pull request merged**